html-tables/html-tables/index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Html-tables</title>
+  </head>
+<body>
+  <h1>Studemt Grade Table</h1>
+  <table>
+    <thead>
+      <tr>
+       <th>Student Name</th>
+       <th>Course</th>
+       <th>Grade</th>
+      </tr>
+     </thead>
+     <tbody>
+      <tr>
+    <td> Grace Hopper</td>
+      <td>Computer Science</td>
+      <td>100</td>
+  </tr>
+  <tr>
+      <td>Alan Turing</td>
```

Comment:
notice how the indentation is shifting here. you want to make sure to keep the indentation even between sibling elements as it makes the code much easier to read.